netgrasp/update/update.py

```diff
@@ -1,149 +1,144 @@
 CURRENT_VERSION = 1
```

Comment:
I think this is now redundant with another define, in __init__ was it in database?